### PR TITLE
Update workflow for api tests

### PR DESCRIPTION
**Pull request by mynguyenlgg**
- Update working-dir
- Update github page link for Allure report regarding the repository name and owner

### CODE REVIEW

**Review by phamthai1912 — APPROVED** [review]

File: .github/workflows/api-regression.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ jobs:
       - name: Run Tests
         continue-on-error: true
         run: |
+          cd api-test/
```
Comment (vantruongt2) [inline .github/workflows/api-regression.yml:31]: Use `working-directory`

File: .github/workflows/api-regression.yml [hunk 2]
```diff
@@ -43,7 +47,7 @@ jobs:
         uses: simple-elf/allure-report-action@v1.7
         if: always()
         with:
-          allure_results: allure-results
+          allure_results: api-test/allure-results
```
Comment (vantruongt2) [inline .github/workflows/api-regression.yml:50]: Why not working-dir?

File: .github/workflows/api-regression.yml (hunk 1 above)
Comment (mynguyenlgg) [reply .github/workflows/api-regression.yml]: I updated

File: .github/workflows/api-regression.yml (hunk 2 above)
Comment (mynguyenlgg) [reply .github/workflows/api-regression.yml]: The working-dir only works with run property, we are using simple-elf/allure-report-action. Therefore, we can not use it in this case.

**Pull request merged**